object_goslice_test.go

```diff
@@ -377,7 +377,7 @@ func TestGoSliceMemUsage(t *testing.T) {
 			// default + default since we don't account for objectGoSlice in (*Object).MemUsage
 			expectedMem: SizeEmptyStruct + SizeEmptyStruct,
 			// overhead + (value + len("length") with string overhead + "length".value + prototype + ints)
-			expectedNewMem: SizeEmptyStruct + (SizeEmptyStruct + (6 + SizeString) + SizeEmptyStruct + (SizeEmptyStruct + SizeEmptyStruct) + SizeNumber*2),
+			expectedNewMem: SizeEmptyStruct + (SizeEmptyStruct + (6 + SizeString) + SizeEmptyStruct + (SizeEmptyStruct + SizeEmptyStruct) + SizeNumber),
```

Comment:
@Gabri3l just wanted to highlight this change. This might not be the best way to address 😬 

```
Unexpected new memory return. Actual: 70 Expected: 78
```

Reply:
I believe this has to do with the prototype. Let me know if you need help digging deeper into this.

Reply:
Ah yeah so when calculating the size of the lazy object prototype ([createArrayProto](https://github.com/mongodb-forks/goja/blob/05a64426fec39209356f96c734bd3831147b8147/builtin_array.go#L1518)) the calculation sort of [double counts](https://github.com/mongodb-forks/goja/blob/f9192da5e15cff88bbe2fe35f834ee5e399ace3d/object_lazy.go#L332) the size (not sure if this was intentional).

When calculating the size of our templated object we get 8 instead of 16.